lib/cc/analyzer/location_description.rb

```diff
@@ -28,23 +25,20 @@ def to_s
 
       attr_reader :location, :suffix
 
-      def render_lines
-        str = location["lines"]["begin"].to_s
-        str << "-#{location["lines"]["end"]}" if location["lines"]["end"]
-        str
+      def render_lines(begin_line, end_line)
+        if end_line != begin_line
```

Comment:
:lipstick: These conditionals should be swapped so that the positive case is first